app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) < 3 or parts[0] != "mv":
+        raise ValueError("Invalid command format. Use"
+                         " 'mv source_path destination_path'.")
+
+    source_path = parts[1]
+    destination_path = parts[2]
+
+    if not os.path.isfile(source_path):
+        raise FileNotFoundError(f"Source file '{source_path}' "
+                                f"does not exist.")
+
+    directory_path = os.path.dirname(destination_path)
+
+    if directory_path and not os.path.exists(directory_path):
+        os.makedirs(directory_path, exist_ok=True)
```

Comment:
Ensure that the function handles the case where the destination path ends with a slash, indicating a directory. You may need to adjust the logic to create the directory if it doesn't exist.